python/lsst/obs/rubinGenericCamera/translator.py

```diff
@@ -0,0 +1,256 @@
+import os
+
+import numpy as np
+import astropy.units as u
+from astropy.time import Time
+
+from astro_metadata_translator import cache_translation
+from lsst.obs.lsst.translators.lsst import LsstBaseTranslator
+
+from lsst.utils import getPackageDir
+
+__all__ = ["StarTrackerNarrwTranslator", "StarTrackerWideTranslator", "StarTrackerFastTranslator",]
+
+
+class RubinGenericCameraTranslator(LsstBaseTranslator):
+    """Metadata translator for Rubin Generic Camera FITS headers"""
+
+    name = None                         # you must specialise this class
+    """Name of this translation class"""
+
+    supported_instrument = None         # you must specialise this class
+    """Supports the LSST Generic Camera instrument."""
+
+    default_search_path = os.path.join(getPackageDir("obs_rubinGenericCamera"), "corrections")
+    """Default search path to use to locate header correction files."""
+
+    default_resource_root = os.path.join(getPackageDir("obs_rubinGenericCamera"), "corrections")
+    """Default resource path root to use to locate header correction files."""
+
+    DETECTOR_MAX = 1
+
+    _const_map = {"detector_num": 0,
+                  "boresight_rotation_coord": "sky",
+                  "physical_filter": "empty",
+                  "detector_group": "None",
+                  "relative_humidity": None,
```

Comment:
Your headers won't have weather information?

Reply:
They don't currently.  Good point, I'll ask what our plans are, but for now this is correct.